### DISCUSSION TITLE
Beelink SER6 7735HS

### DISCUSSION THREAD

**Discussion by akidarsa** [isc30/ryzen-gpu-passthrough-proxmox#9] [General]
Hi All,

Followed the guide but still getting Code43 inside Windows 11.
Wondering if anyone is willing to help to diagnose why…

CPU: 16 x AMD Ryzen 7 7735HS with Radeon Graphics (1 Socket)

Configuration:

> agent: 1
> args: -cpu 'host,-hypervisor,kvm=off'
> bios: ovmf
> boot: order=ide0;net0
> cores: 8
> cpu: host
> efidisk0: local-lvm:vm-800-disk-0,efitype=4m,pre-enrolled-keys=1,size=4M
> hostpci0: 0000:74:00.0,pcie=1,romfile=vbios_7735HS.bin,x-vga=1
> hostpci1: 0000:74:00.1,pcie=1,romfile=AMDGopDriver.rom
> ide0: local-lvm:vm-800-disk-1,discard=on,size=128G,ssd=1
> ide2: none,media=cdrom
> machine: pc-q35-8.0
> memory: 12288
> meta: creation-qemu=8.0.2,ctime=1697460714
> name: JumpHost
> net0: virtio=F6:F7:8E:28:90:D5,bridge=vmbr0,firewall=1
> net1: e1000=26:32:78:BD:E8:08,bridge=vmbr1,firewall=1
> numa: 0
> onboot: 1
> ostype: win11
> parent: WorkingConfig
> scsihw: virtio-scsi-single
> smbios1: uuid=99b5c3ea-be4b-4d28-ab0b-f3fefa7a3d96
> sockets: 1
> tpmstate0: local-lvm:vm-800-disk-2,size=4M,version=v2.0
> vga: none
> vmgenid: ff02d2ed-4f27-46f2-b92a-76e573d57d5d

**Comment by isc30** [comment]
Hi, did all the steps in the guide succeed? Were you able to install the GPU drivers properly?

**Reply by akidarsa** [reply]
Hi, apologies on the delay of my reply.

The AMD GPU installed inside Windows, I believe it's installed correctly.

<img width="782" alt="Screenshot 2023-12-13 at 14 11 22" src="https://github.com/isc30/ryzen-7000-series-proxmox/assets/1009362/f9496374-da44-45ed-b680-85a72b99f72b">
<img width="606" alt="Screenshot 2023-12-13 at 14 11 36" src="https://github.com/isc30/ryzen-7000-series-proxmox/assets/1009362/fff7854d-c115-4801-9cf4-e26960bc26cf">

**Reply by isc30** [reply]
did it let you install the drivers? did it work with win10 and seabios?

**Reply by akidarsa** [reply]
I was hoping to use it on my Windows 11 System on UEFI.
I guess I will try to do it on Win 10 with SEABIOS see if it works.
Will update here.

**Reply by akidarsa** [reply]
Win 10 works. I guess there’s some issues with Win 11…

**Reply by isc30** [reply]
Hey! okay, great, that's a good sign. It means 2 crucial things:

- The `host` (proxmox server) configuration is correct and can perform the passthrough without crashing.
- The extracted `vbios` is working properly.

I would follow these progressive steps to check where its failing:
- Look into passing the audio device now to win10 with the `AMDGopDriver` and see if that still works.
- Try creating another win10 VM with UEFI bios this time, and check if it works (gpu+audio).

If the Win10 VM with UEFI works but Win11 with the same qemu config doesn't, then its *very* strange

**Reply by akidarsa** [reply]
Ah, Thank you for the diagnostics steps. Let me try and update here. Apologies for the delays, a bit busy with work.
